index.js

```diff
@@ -18,8 +18,8 @@ function collectOptions (val, memo) {
   } else {
     console.log('Error: Option \'%s\' is invalid.', val)
     console.log('Please provide options in this way:')
-    console.log('-O [Key]" + coapOptionSeperator + "[Value]')
-    console.log('OR --coap-option [Key]' + coapOptionSeperator + '[Value]')
+    console.log('-O 2048,HelloWorld')
```

Comment:
replacing the seperator `,` with  `' + coapOptionSeperator + '` would keep the help-text upToDate when the seperator is exchanged

Reply:
Done